planner/core/src/java/seaclouds/planner/WebServiceLayer.java

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package seaclouds.planner;
+
+import seaclouds.utils.toscamodel.*;			// parser
+
+/* servlet */
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+/* json parser */
+import org.json.simple.parser.*;
+import org.json.simple.*;
+
+/* Map */
+import java.util.*;
+
+
+public class WebServiceLayer extends HttpServlet {
```

Comment:
Maybe not for this release, but take into account that using Servlets it's a very outdated way to build a REST API. Its way easier to use Jersey + Jackson as we use in https://github.com/adriannieto/SeaCloudsPlatform/blob/feature/dropwizard/dashboard/src/main/java/eu/seaclouds/platform/dashboard/resources/DeployerResource.java which eases a lot the handling of the parameters and repsonses.